### PR TITLE
"Errors" upgrade docs

### PR DESCRIPTION
**Pull request by millette**
Based on https://github.com/caddyserver/caddy/issues/3336 discussion.

### CODE REVIEW

File: src/docs/markdown/v2-upgrade.md [hunk 1]
```diff
@@ -146,6 +147,46 @@ browse /subfolder/
...
+
+Custom error pages can be accomplished with [`handle_errors`](/docs/caddyfile/directives/handle_errors).
+
+
+- **v1:**:
+
+```
+errors {
+	404 404.html # Not Found
+	500 500.html # Internal Server Error
```
Comment (mholt) [inline src/docs/markdown/v2-upgrade.md]:
```suggestion
	404 404.html
	500 500.html
```

The comments don't add anything important here, and are distracting, so let's take them out.

File: src/docs/markdown/v2-upgrade.md [hunk 2]
```diff
@@ -146,6 +147,46 @@ browse /subfolder/
...
+}
+```
+
+or
+
+```
+errors {
+	* default_error.html
+}
+```
```
Comment (mholt) [inline src/docs/markdown/v2-upgrade.md]: Remove this second example, it becomes confusing what the transition is supposed to go from.

File: src/docs/markdown/v2-upgrade.md [hunk 3]
```diff
@@ -146,6 +147,46 @@ browse /subfolder/
...
+
+or even
+
+```
+handle_errors {
+	rewrite * /{http.error.status_code}
+	reverse_proxy https://http.cat
+}
+```
```
Comment (mholt) [inline src/docs/markdown/v2-upgrade.md]: Similarly, remove this second example. It doesn't really add anything, and makes the transition confusing.

**Review by mholt — CHANGES_REQUESTED** [review]
Thank you!

**Comment by millette** [comment]
There you go. You're welcome. Thanks for caddy and making v2 100% libre :-)

**Review by mholt — APPROVED** [review]
Looks great now!

And you're welcome -- I'm grateful to sponsors that make it possible.

**Pull request merged**